.pre-commit-config.yaml

```diff
@@ -1,8 +1,8 @@
 repos:
 
 # Automatically sort imports
-- repo: https://github.com/PyCQA/isort.git
-  rev: 5.5.3
+- repo: https://github.com/PyCQA/isort
+  rev: 5.6.4
```

Comment:
Was there a reason this was updated also, or just an opportunity to update? Why to 5.6.4?

Reply:
Just an upportunity update to the most recent version that supports pre-commit <2.9, and that is also already referenced in other repositories such as ISPyB, zocalo, workflows, dlstbx, ...